app/core/simulation.ts

```diff
@@ -226,19 +226,6 @@ export default class Simulation extends Schema implements ISimulation {
     this.applyPostEffects()
   }
 
-  getCurrentBattleResult(playerId: string) {
```

Comment:
this method is used nowhere in the codebase so i removed it. Please double check with me

Reply:
indeed, seems to be legacy  code